### ISSUE TITLE
Make some of the demos in HPC-GAP into tests.

### ISSUE THREAD

**Issue by markuspf** [gap-system/gap#218]
Most of the examples in the demos/ folder either don't work, or do not work in an obvious way.

Make some meaningful demos, and also use them as tests.

**Comment by alex-konovalov** [comment]
Agree. For the moment, `demo/demo.g` is known to work and what is used in regression tests with Jenkins CI - I suggest to run it if you make any changes and would like to test them.

**Comment by alex-konovalov** [comment]
@ChrisJefferson I suggest that I will create new directory `tst/hpcgap` and start to move some of the tests from `demo` directory there, starting from those that we run as a part of HPC-GAP tests with Jenkins.
